src/bank.ts

```diff
@@ -57,7 +57,7 @@ export class Bank extends ServerNode {
   async addBlocks(balanceLock: string, transactions: Transaction[], account: Account) {
     return await this.postData(
       "/blocks",
-      { account, balanceLock, transactions }
+      account.createSignedData({ balanceLock, transactions })
```

Comment:
Oh my gosh, so sorry.  I meant to say use the `account.createBlockMessage` method which doesn't require the destructuring (just feed the arguments in that order).  Very sorry for the confusion.